control-plane/agents/src/common/errors.rs

```diff
@@ -688,6 +690,12 @@ impl From<SvcError> for ReplyError {
                 source,
                 extra,
             },
+            SvcError::InvalidLabel { .. } => ReplyError {
+                kind: ReplyErrorKind::InvalidArgument,
+                resource: ResourceKind::Unknown,
```

Comment:
Use the same resource kind here as per above suggestion. 

```suggestion
                SvcError::InvalidLabel { resource_kind, .. } => ReplyError {
                       kind: ReplyErrorKind::InvalidArgument,
                       resource: resource_kind,
```